New version: OpenAI v0.12.0

JuliaRegistrator opened the pull request:
- Registering package: OpenAI
- Repository: https://github.com/JuliaML/OpenAI.jl
- Created by: @roryl23
- Version: v0.12.0
- Commit: 5443403a5053d2d320b5a8bbb00ca359c0434018
- Reviewed by: @roryl23
- Reference: https://github.com/JuliaML/OpenAI.jl/commit/5443403a5053d2d320b5a8bbb00ca359c0434018#commitcomment-165483859
- Description: OpenAI API wrapper for Julia
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

- Integrated StreamCallbacks.jl
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd67c24247c916934cb7946cd806f73a6e5d8948210e0a79edd5450e229753116e4e0d2d8211f0799c4b17cdc8315714a724be6351190af0c0bbe99254d6aa70da356bdc410246077609c47510f07648e45812fbb23c6e943d8b72f853c05b3e67a53b8ecfb5bd70c36d5cc2123fdbb15c9866f662edf697f7bd24a47ef50d0dad794cf49a5256d34c368b62d2d40a15c89f98ee238f2924d73d195f6b7ff4567663111bc97aee882be7f815893e3f8449f -->

roryl23 commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- Integrated StreamCallbacks.jl

JuliaRegistrator committed:
New version: OpenAI v0.12.0

UUID: e9f21f70-7185-4079-aca2-91159181367c
Repo: https://github.com/JuliaML/OpenAI.jl.git
Tree: ecabeb2187974065acaf8c273f177d3e70e267d0

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
